README.md

```diff
@@ -477,3 +483,37 @@ backslash. The placeholders have the following meaning:
 * `%S`: print current stack of virtual machine
 * `%e`: print current environment of bindings
 * `%m`: print current store of metavariable solutions
+
+## Executing actors
+
+Actors are executed using the `exec` command, in the context of all previous declarations and definitions. After the actors have finished running, a "typing derivation" is extracted and printed on the screen. For example, running the actor
+```
+exec  check@p. 'Check?t.
+   p! ['Arr 'Nat 'Nat].
+   p! ['Lam \z. ['Emb
+         ['App ['Rad ['Lam \w. ['Emb w]] ['Arr 'Nat 'Nat]]
+         ['Emb z]]]].
```

Comment:
It looks like `'Check?t.` isn't doing anything here? (or am I misinterpreting it?)

Reply:
Indeed it is a (harmless) leftover from something else. Thanks for the push to simply remove it, I will do so now on the main branch!